test/Microsoft.AspNetCore.SignalR.Tests/Microsoft.AspNetCore.SignalR.Tests.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```

Comment:
This does not seem right. Wouldn't the second TargetFrameworks overwrite the first and you would effectively always end up with `netcoreapp2.0;net461`? Note the order in: https://github.com/aspnet/SignalR/pull/486/files#diff-8a366c3f3e35af025106ceaa9af5f7afR6